otter/test/convergence/test_gathering.py

```diff
@@ -605,7 +604,7 @@ def test_lb_disappeared_during_feed_fetch(self):
         eff = get_clb_contents()
         self.assertEqual(
             perform_sequence(seq, eff),
-            ([assoc_obj(CLBNode.from_node_json(2, node21), drained_at=2.0)],
+            ([attr.assoc(CLBNode.from_node_json(2, node21), _drained_at=2.0)],
```

Comment:
Huh. That's an interesting bug in `assoc`; I thought it would have behaved the same as `__init__` when it came to private variables.